src/components/friends/FriendsPage.jsx

```diff
@@ -14,62 +15,72 @@ class FriendsPage extends Component {
     this.props = props;
     this.state = {
       friendsList: [],
+      loading: true,
     };
     this.loadFriends();
   }
 
+  handleUnFriend = (item) => {
+    friendsService.unFriend(item).then((success) => {
+      if (success) {
+        this.loadFriends();
+      }
+    }).catch((error) => {
+      // eslint-disable-next-line no-alert
+      alert(error);
+    });
+  }
+
   loadFriends() {
     const friendsList = [];
-    const userID = localStorage.getItem("userID");
-    friendsService.getAuthorFriends(userID).then((response) => {
+    const { user } = this.props;
+    friendsService.getAuthorFriends(user.id).then((response) => {
       for (let i = 0; i < response.length; i += 1) {
         const newFriend = {};
         const friend = response[i];
 
-        newFriend.name = friend.displayName;
+        newFriend.displayName = friend.displayName;
```

Comment:
is there a reason we create a `newFriend` object instead of just using the `friend`?

Reply:
@zhaoyuxuan wrote this part in #119 . I just changed the field name here so we can pass the friend object to other pages. Do you want to use `friend` directly?

Reply:
Yea so in that PR we were changing the property names. Like ex: we changed displayName to name
`newFriend.name = friend.displayName;` but if we are using the same properties then we could just use the original object. But it's a super small thing we can just leave it the way it is.

Reply:
Just updated it to use the friend object directly. For the final milestone i'll also rename the username props to displayName in front end, so that we could be more consistent with the db.